edit-post/components/header/keyboard-shortcuts-help-menu-item/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External Dependencies
+ */
+import { flow } from 'lodash';
+
+/**
+ * WordPress Dependencies
+ */
+import { withSelect, withDispatch } from '@wordpress/data';
+import { displayShortcut } from '@wordpress/keycodes';
+
+/**
+ * WordPress Dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { compose } from '@wordpress/compose';
+import { MenuItem } from '@wordpress/components';
+
+const MODAL_NAME = 'edit-post/keyboard-shortcut-help';
+
+export function KeyboardShortcutsHelpMenuItem( { toggleModal, onSelect } ) {
+	return (
+		<MenuItem
+			onClick={ flow( onSelect, toggleModal ) }
+			shortcut={ displayShortcut.access( 'h' ) }
+		>
+			{ __( 'Keyboard Shortcuts' ) }
+		</MenuItem>
+	);
+}
+
+export default compose( [
+	withSelect( ( select ) => ( {
+		isModalActive: select( 'core/edit-post' ).isModalActive( MODAL_NAME ),
```

Comment:
Do we need this? When would you be able to close the modal using this button?

Reply:
very true - removed the toggle functionality, it's now just an opener.